### ISSUE TITLE
Lookup for ConfigMap "enableapigroupversions" hardcodes "velero" namespace

### ISSUE THREAD

**Issue by puneetkatyal** [vmware-tanzu/velero#4532]
**What steps did you take and what happened:**
[A clear and concise description of what the bug is, and what commands you ran.)
* Install velero in the non-default namespace (https://velero.io/docs/v1.7/customize-installation/#install-in-any-namespace) and specify feature flag `--features=EnableAPIGroupVersions.`
* Create a ConfigMap called "enableapigroupversions" in this custom namespace (https://velero.io/docs/v1.7/enable-api-group-versions-feature/#advanced-procedure-for-customizing-the-version-prioritization) 
* Velero is unable to find this configmap because it's hardcoded to look for it in the default "velero" namespace.
* I'm using v1.7.0 but looking at the code, the issue exists in v1.7.1 too.
https://github.com/vmware-tanzu/velero/blob/v1.7.1/pkg/restore/prioritize_group_version.go#L224

**What did you expect to happen:**
Velero should look for this configmap in the custom namespace

**The following information will help us better understand what's going on**:

_If you are using velero v1.7.0+:_  
Please use `velero debug  --backup <backupname> --restore <restorename>` to generate the support bundle, and attach to this issue, more options please refer to `velero debug --help` 

_If you are using earlier versions:_  
Please provide the output of the following commands (Pasting long output into a [GitHub gist](https://gist.github.com) or other pastebin is fine.)
- `kubectl logs deployment/velero -n velero`
- `velero backup describe <backupname>` or `kubectl get backup/<backupname> -n velero -o yaml`
- `velero backup logs <backupname>`
- `velero restore describe <restorename>` or `kubectl get restore/<restorename> -n velero -o yaml`

- `velero restore logs <restorename>`
```
time="2022-01-15T01:49:57Z" level=debug msg="No enableapigroupversion config map found in velero namespace. Using pre-defined priorities." logSource="pkg/restore/prioritize_group_version.go:198" restore=cluster-services/pk-restore-ns01-20220114174956
```

**Anything else you would like to add:**
[Miscellaneous information that will assist in solving the issue.]

**Environment:**

- Velero version (use `velero version`): v1.7.0
- Velero features (use `velero client config get features`): EnableAPIGroupVersions
- Kubernetes version (use `kubectl version`):
- Kubernetes installer & version:
- Cloud provider or hardware configuration:
- OS (e.g. from `/etc/os-release`):

**Vote on this issue!**

This is an invitation to the Velero community to vote on issues, you can see the project's [top voted issues listed here](https://github.com/vmware-tanzu/velero/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc).  
Use the "reaction smiley face" up to the right of this comment to vote.

- :+1: for "I would like to see this bug fixed as soon as possible"
- :-1: for "There are more important bugs to focus on right now"

**Comment by blackpiglet** [comment]
I think this should be fixed in #4346 
Could you try with main version of velero to test whether it works? 
We are going to cut velero 1.8 release recently. This fix is included in release 1.8

**Comment by puneetkatyal** [comment]
@blackpiglet I'll verify this with the main branch today. It does look like it's fixed with #4346.

**Comment by puneetkatyal** [comment]
I'm able to load the configmap using the `main` branch version. Feel free to close this issue.
